### ISSUE TITLE
Won't install in Pale Moon

### ISSUE THREAD

**Issue by hook321** [downthemall/downthemall-legacy#254]
The last version that I am aware of being installable in Pale Moon is version 2.0.18.

**Comment by ianamk** [comment]
+1
with the coming of FF57 I've been migrating my bookmarks/settings/addons to PaleMoon any chance for a new version that installs and runs on current PaleMoon
http://developer.palemoon.org/Add-ons:Site
http://developer.palemoon.org/Add-ons:Extensions/Jetpack

**Comment by Ferretaa** [comment]
I'd also think it would be great if the latest version worked with PaleMoon - with the changes happening to Firefox I have started to use PaleMoon instead. I think a lot of the people that use DTA would possibly switch to PaleMoon if they found out it was being actively supported as a platform.

**Comment by CybertronicToon** [comment]
I agree. I have version 2.0.18 of DownThemAll installed on Pale Moon, but it would be nice to have the newest version, and official support for it. Especially considering full DTA will never work on firefox 57+.

**Comment by hook321** [comment]
I would recommend installing Waterfox instead.

**Comment by ghost** [comment]
Version 3.0.8 works with Pale Moon 28.
